standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-3.2.0-to-4.0.0-alpha-1.mysql.sql

```diff
@@ -153,6 +153,9 @@ ALTER TABLE COMPACTION_QUEUE ADD CQ_COMMIT_TIME bigint;
 -- HIVE-24770
 UPDATE SERDES SET SLIB='org.apache.hadoop.hive.serde2.MultiDelimitSerDe' where SLIB='org.apache.hadoop.hive.contrib.serde2.MultiDelimitSerDe';
 
+-- HIVE-27180
```

Comment:
we have already released 4.0.0-alpha2. So this change should go into the "4.0.0-alpha-2-to-4.0.0" upgrade script

Reply:
Will incorporate this change.